pkg/driver/distro/flatcar.go

```diff
@@ -94,9 +122,7 @@ func (f *flatcar) customizeBuild(ctx context.Context,
 		return nil, nil
 	}
 	printer.Logger.Info("Flatcar detected; relocating kernel tools.", printer.Logger.Args("version", f.versionID))
-	out, err := exec.CommandContext(ctx, "/bin/bash", "-c", flatcarRelocateScript).Output()
-	if err != nil {
-		printer.DefaultText.Print(string(out))
-	}
+	out, err := exec.CommandContext(ctx, "/bin/bash", "-c", flatcarRelocateScript).CombinedOutput()
+	printer.DefaultText.Print(string(out))
```

Comment:
I'd leave it like it was before; no need to be verbose if no error happens (unformatted output is ugly :/ )

Reply:
Done.